pkg/loader/compose/v1v2.go

```diff
@@ -109,24 +109,26 @@ func loadPorts(composePorts []string) ([]kobject.Ports, error) {
 		// Split up the ports / IP without the "/tcp" or "/udp" appended to it
 		justPorts := strings.Split(protocolCheck[0], character)
 
-		if len(justPorts) == 3 {
+		if len(justPorts) >= 3 {
 			// ex. 127.0.0.1:80:80
+			// ex. fe80::6ec2:17ff:fe84:4270:8096:8096
 
+			l := len(justPorts)
 			// Get the IP address
-			hostIP := justPorts[0]
+			hostIP := protocolCheck[0][:len(protocolCheck[0])-2-(len(justPorts[l-2])+len(justPorts[l-1]))]
```

Comment:
This seems a bit hacky IMO... 

Any way to use a library or a put this in a separate function to make sure we get just the IP address?